api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -620,14 +621,20 @@ def _resolve_grains(self, nodes: List[Node], ssh_client):
     def _prepare_roster(
         self, nodes: List[Node], priv_key, roster_path
     ):
-        roster = {
-            node.minion_id: {
+        tmp_dir = os.getenv('TMPDIR')
+        thin_dir = (Path(tmp_dir).resolve() / 'salt_thin_dir') if tmp_dir else None
```

Comment:
yes we have used user home dir 

https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/api/python/provisioner/config.py#L48

Reply:
@ypise @mazinamdar `thin_dir` is a different thing, it just might be specified in roster but it's a kind of utility temporary directory which salt ssh [uses](https://docs.saltproject.io/en/latest/topics/ssh/roster.html) to prepare remote commands. By default salt creates it in some global location (e.g. in `/var/tmp`) and it may grow to bit sizes (in case of ISO replication: one ISO copy peer each node). So makes sense to have some control over that.

Reply:
Sure, got it.
I was actually thinking if we need `--wipe` (https://docs.saltproject.io/en/latest/ref/cli/salt-ssh.html#cmdoption-salt-ssh-w)

Reply:
Created a follow-up task https://jts.seagate.com/browse/EOS-19068